### PR TITLE
fix: add search by nft and shorten search placehodler

### LINKED ISSUE #338: Input gets zoomed upon initialization of the page
### Context information

application version: 
device:  Iphone 12
browser: Chrome
operating system: 

### Steps to reproduce

https://github.com/aeternity/aescan/assets/11598532/fbd858f2-d6b7-4f58-b70b-6b91f98c3c03

### LINKED ISSUE #420: Search field placeholder text is too long
### Context information

application version: I can't find it at https://aescan.io/
browser: Google Chrome@116

### Steps to reproduce
- open https://aescan.io/

### What is expected?
To see the whole placeholder text

### What is actually happening?
Screenshot 2023-08-02 at 13 50 46

### LINKED ISSUE #492: Search by NFT

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #420 
fixes #338 
fixes #492 

## Demo

https://github.com/aeternity/aescan/assets/15363559/818e06ef-9e83-46bb-a02e-00a21df3dd69

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] fix nft hints
- [x] add search by nft
- [x] shorten placeholder

### CODE REVIEW

File: src/utils/hints/nftsHints.js [hunk 1]
```diff
@@ -1,7 +1,23 @@
 export const nftsHints = {
```
Comment (janmichek) [inline src/utils/hints/nftsHints.js:1]: There were 2 files for NFT hints. Probably product of some merging. I united it into just one. Hence every nft hint should also be checked @Liubov-crypto

File: src/components/TheSearchBar.vue [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <input
       v-model="query"
       class="search-bar__input"
-      placeholder="Search accounts, transactions, names, contracts, oracles, state channels, keyblocks, and microblocks"
+      placeholder="Search anything on aeScan"
```
Comment (janmichek) [inline src/components/TheSearchBar.vue:6]: please check the zooming issue on iPhone @Liubov-crypto

Comment (Liubov-crypto) [reply src/components/TheSearchBar.vue:6]: > please check the zooming issue on iPhone @Liubov-crypto

Still reproducible for me on iPhone 13 in Safari browser ver. 17.1.2

**Review by Liubov-crypto — CHANGES_REQUESTED** [review]
It seems that search for NFT by name is not accurate. For example, if I'm searching for 'cool' nft I will receive only 1 result for nft, but we have 2 nft with such name - cool cat and coolcat. 

@janmichek please check it: 

https://github.com/aeternity/aescan/assets/69896204/742a33d5-8a93-4e80-8470-b4173693895c

Also, zooming on iphone still reproducible for me.

File: src/components/TheSearchBar.vue (hunk 2 above)
Comment (janmichek) [reply src/components/TheSearchBar.vue:6]: I tried to apply possible fix. Please try again now

**Comment by janmichek** [comment]
@Liubov-crypto 

> It seems that search for NFT by name is not accurate. For example, if I'm searching for 'cool' nft I will receive only 1 result for nft, but we have 2 nft with such name - cool cat and coolcat.

That's a known bug. The search is case sensitive currently. The fix is requested from mdw

**Comment by janmichek** [comment]
Build stopped working for unknown reason

**Comment by janmichek** [comment]
Moved here https://github.com/aeternity/aescan/pull/626

**Pull request closed without merging**